packages/cli/internal/pkg/cli/logs_core.go

```diff
@@ -124,6 +124,19 @@ func (o *logsSharedOpts) followLogGroup(logGroupName string) error {
 }
 
 func (o *logsSharedOpts) displayEventFromChannel(channel <-chan cwl.StreamEvent) error {
+	firstEvent := <-channel
+
+	if firstEvent.Err != nil {
+		return firstEvent.Err
+	}
+	if len(firstEvent.Logs) > 0 {
+		for _, line := range firstEvent.Logs {
+			printLn(line)
+		}
+	} else {
+		printLn("There are no new logs. Please wait for new logs to show...")
+	}
+
```

Comment:
This replicates the existing logic below, we only need to conditionally show a message in the else block of the first event, can we simplify that with a flag?

```
var firstEvent = true
....
   if (firstEvent) {
      printLn("There are no new logs. Please wait for new logs to show...")
   } else {
      log.Debug().Msg("No new logs")
   }
...
firstEvent = false
```